assets/xml/objects/object_um.xml

```diff
@@ -49,31 +54,35 @@
-        <Limb Name="object_um_Standardlimb_011D38" Type="Standard" EnumName="OBJECT_UM_LIMB_0D" Offset="0x11D38" />
-        <Limb Name="object_um_Standardlimb_011D44" Type="Standard" EnumName="OBJECT_UM_LIMB_0E" Offset="0x11D44" />
-        <Limb Name="object_um_Standardlimb_011D50" Type="Standard" EnumName="OBJECT_UM_LIMB_0F" Offset="0x11D50" />
-        <Limb Name="object_um_Standardlimb_011D5C" Type="Standard" EnumName="OBJECT_UM_LIMB_10" Offset="0x11D5C" />
-        <Limb Name="object_um_Standardlimb_011D68" Type="Standard" EnumName="OBJECT_UM_LIMB_11" Offset="0x11D68" />
-        <Limb Name="object_um_Standardlimb_011D74" Type="Standard" EnumName="OBJECT_UM_LIMB_12" Offset="0x11D74" />
-        <Limb Name="object_um_Standardlimb_011D80" Type="Standard" EnumName="OBJECT_UM_LIMB_13" Offset="0x11D80" />
-        <Limb Name="object_um_Standardlimb_011D8C" Type="Standard" EnumName="OBJECT_UM_LIMB_14" Offset="0x11D8C" />
-        <Limb Name="object_um_Standardlimb_011D98" Type="Standard" EnumName="OBJECT_UM_LIMB_15" Offset="0x11D98" />
-        <Skeleton Name="object_um_Skel_011DF8" Type="Flex" LimbType="Standard" LimbNone="OBJECT_UM_LIMB_NONE" LimbMax="OBJECT_UM_LIMB_MAX" EnumName="object_um_Limbs" Offset="0x11DF8" />
-        <Animation Name="object_um_Anim_01213C" Offset="0x1213C" />
-        <Animation Name="object_um_Anim_0126C4" Offset="0x126C4" />
-        <Animation Name="object_um_Anim_012CC0" Offset="0x12CC0" />
+
+        <Limb Name="gUmWagonSeatAndShaftsLimb" Type="Standard" EnumName="UM_LIMB_SEAT_AND_SHAFTS" Offset="0x11CA8" /> <!-- Chair and donkey attachment -->
+        <Limb Name="gUmWagonRootLimb" Type="Standard" EnumName="UM_LIMB_WAGON_ROOT" Offset="0x11CB4" />
+        <Limb Name="gUmWagonRightWheelLimb" Type="Standard" EnumName="UM_LIMB_WAGON_RIGHT_WHEEL" Offset="0x11CC0" />
+        <Limb Name="gUmWagonLeftWheelLimb" Type="Standard" EnumName="UM_LIMB_WAGON_LEFT_WHEEL" Offset="0x11CCC" />
+        <Limb Name="gUmWagonCartBedLimb" Type="Standard" EnumName="UM_LIMB_WAGON_CART_BED" Offset="0x11CD8" />
+        <Limb Name="gUmWagonHandBrakeLimb" Type="Standard" EnumName="UM_LIMB_WAGON_HAND_BRAKE" Offset="0x11CE4" />
+        <Limb Name="gUmWagonCartCoverLimb" Type="Standard" EnumName="UM_LIMB_WAGON_CART_COVER" Offset="0x11CF0" />
+        <Limb Name="gUmWagonLeftLeafSpringLimb" Type="Standard" EnumName="UM_LIMB_WAGON_LEFT_LEAF_SPRING" Offset="0x11CFC" />
+        <Limb Name="gUmWagonRightLeafSpringLimb" Type="Standard" EnumName="UM_LIMB_WAGON_RIGHT_LEAF_SPRING" Offset="0x11D08" />
+        <Limb Name="gUmWagonSteeringWheelLimb" Type="Standard" EnumName="UM_LIMB_WAGON_STEERING_WHEEL" Offset="0x11D14" />
+        <Limb Name="gUmCremiaRootLimb" Type="Standard" EnumName="UM_LIMB_CREMIA_ROOT" Offset="0x11D20" />
+        <Limb Name="gUmCremiaThoraxLimb" Type="Standard" EnumName="UM_LIMB_CREMIA_THORAX" Offset="0x11D2C" />
```

Comment:
I called this her "Torso" in my ma2 PR. I can see the argument for "Thorax" or "Chest", though. It would be good to be consistent here, since it's the same limb basically. What do you think?